### PR TITLE
Gives Magus (Nadir) a lighting cycle like Earth and Abzu do

### PR DESCRIPTION
**Pull request by TheTyrant25**
[FEAT]
## About the PR
This gives Magus an 8 hour rotational cycle, bathing it in Shidd's light half the time, and Fugg's light the other half of the time, with gradual changes between the two 'noon's.
### lore warning
After some discussion with Cogwerks and Kubius in #imlore, it was decided that despite Magus orbiting Typhon, the majority of its light comes from the two stars in the Frontier System instead, as it orbits out quite far (in the royal rings district). Shidd is a Red Giant, and Fugg is a blue star, so despite nadir itself sitting in a very deep trench and the oceans above it filtering out most light, some still makes it through, giving a very subtle effect.

## Why's this needed? 
The other terrestrial locations (e.g. Oshan and Centcomm) had this feature, so it makes sense to have it. Also the purple tones when Shidd is high in the sky is quite pretty.
## Changelog
```changelog
(u)Tyrant
(+)Nadir now has a solar cycle, like Oshan does.
```

### CODE REVIEW

**Comment by M-Earthfire** [comment]
Is aqua tenebrae non-dense enough to let light down to the bottom of an ocean?

Or is Nadir just a few metres under the acid ocean.

**Comment by RelentlessGarbage** [comment]
Not opposed to this per-se, this would make Nadir's acid sea prettier. However, given Nadir's storminess and the reduced sunlight that would come from the clouds, perhaps adding occasional random periods of darkness (corresponding to heightened storm activity) would be cool. Might make exploration slightly more hazardous on occasion.

File: _std/defines/time.dm [hunk 1]
```diff
@@ -10,32 +10,54 @@
...
+	// shidd and fugg are on opposite sides of the sky, so as one sets, the other rises.
+	// #define OCEAN_LIGHT  rgb(0,0,50)
+	#if (BUILD_TIME_HOUR == 0) || (BUILD_TIME_HOUR - 8 == 0) || (BUILD_TIME_HOUR - 16 == 0)
+		#define OCEAN_LIGHT  rgb(0.10 * 128, 0.10 * 170, 1.00 *  75, 0.65 * 255)
+	#elif (BUILD_TIME_HOUR == 1) || (BUILD_TIME_HOUR - 12 == 1) || (BUILD_TIME_HOUR - 16 == 1)
+		#define OCEAN_LIGHT  rgb(0.10 *  64, 0.10 * 191, 1.00 *  88, 0.65 * 255)
+	#elif (BUILD_TIME_HOUR == 2) || (BUILD_TIME_HOUR - 12 == 2) || (BUILD_TIME_HOUR - 16 == 2)
+		#define OCEAN_LIGHT  rgb(0.10 *   0, 0.10 * 255, 1.00 * 100, 0.65 * 255) // noon (fugg) rgb(0,26,100), quite bluey
+	#elif (BUILD_TIME_HOUR == 3) || (BUILD_TIME_HOUR - 12 == 3) || (BUILD_TIME_HOUR - 16 == 3)
+		#define OCEAN_LIGHT  rgb(0.10 *  64, 0.10 * 191, 1.00 *  88, 0.65 * 000)
```
Comment (Kubius) [inline _std/defines/time.dm:25]: Why is there 0.65 * 000 on this one line?

**Review by Kubius — COMMENTED** [review]
Not 100% sure how I feel about the particulars, but I like the concept as a whole and I don't have any specific objections that should stop this from getting in (aside from one weird thing I don't quite get).

**Comment by Kubius** [comment]
In reply to the thoughts of heightened storm activity - at some point, I'd like to see a Nadir-specific surface storm event that disrupts offsite comms and transception, and occasionally chucks a big acid hailstone down at the breach hull. (Might end up making that myself if I can figure out how.)

File: _std/defines/time.dm (hunk 1 above)
Comment (TheTyrant25) [reply _std/defines/time.dm:25]: oh that isn't supposed to be like that, whoops

**Review by ZeWaka — APPROVED** [review]

**Review by Kubius — APPROVED** [review]

**Pull request merged**